### PR TITLE
misc: Add bug report template

### PR DESCRIPTION
**Pull request by BobbyRBruce**
Change-Id: I3acf7a1991f889462c0f2604d251dead563846c2

### CODE REVIEW

**Comment by BobbyRBruce** [comment]
I believe this needs to be on the stable branch for this to work. To keep the branches in-sync I'd like to merge into develop and cherry pick this change when I create a minor release.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+**gem5 Modifications**
+If you have modified gem5 in some way, please state, to the best of your ability, how it has been modified.
+
+**To Reproduce**
+Steps to reproduce the behavior. Please assuming starting from a clean repository:
+1. Compile gem5 with command ...
+2. Execute the simulation with...
+
+**Terminal Output**
+If applicable, add the terminal output here. If long, only include the relevant lines.
```
Comment (powerjg) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: Can you add something here about *not* attaching screenshots. Maybe providing "```" for the user to format correctly?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+**Affects version**
+State which version of gem5 this bug was found in. If on the develop branch state the Commit revision ID you are working.
+
+**gem5 Modifications**
+If you have modified gem5 in some way, please state, to the best of your ability, how it has been modified.
+
+**To Reproduce**
+Steps to reproduce the behavior. Please assuming starting from a clean repository:
+1. Compile gem5 with command ...
+2. Execute the simulation with...
```
Comment (powerjg) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: I would add something here about including the python script as well as the parameters that was used.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (BobbyRBruce) [reply .github/ISSUE_TEMPLATE/bug_report.md:40]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (BobbyRBruce) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: Done

**Review by powerjg — APPROVED** [review]

**Pull request merged**